.circleci/config.yml

```diff
@@ -55,6 +55,12 @@ jobs:
       - run:
           name: Project Tests
           command: yarn test:projects:cra-ts
+      - run:
+          name: Danger JS
```

Comment:
would suggest some other name that will provide the intent of this step, like 'Conventions test'

Reply:
I would rather leave it as it is (for now). I intentionally used that name to make it clear that the step is driven by `dangerfile.ts`.

Reply:
lets discuss offline - cannot agree here, as there are other steps that are driven by dedicated tools (like Screener), but so far we have used semantics-based approach for naming.